src/Device/Driver/OpenVario.cpp

```diff
@@ -46,65 +46,162 @@ class OpenVarioDevice : public AbstractDevice {
   bool PutBugs(double bugs, OperationEnvironment &env) override;
   void OnCalculatedUpdate(const MoreData &basic,
                   const DerivedInfo &calculated) override;
-  static bool POV(NMEAInputLine &line, NMEAInfo &info);
+private:
+  bool POV(NMEAInputLine &line, NMEAInfo &info);
+  bool ComposeWrite(const char p_type,PolarCoefficients &p,
+  		  OperationEnvironment &env);
+  bool PutIdealPolar(const DerivedInfo &calculated,
+  		  OperationEnvironment &env);
+  bool PutRealPolar(const DerivedInfo &calculated,
+  		  OperationEnvironment &env);
+  bool RepeatBugs(OperationEnvironment &env);
+  bool RepeatBallast(OperationEnvironment &env);
+  bool RepeatMacCready(OperationEnvironment &env);
+  bool IsEqual(const PolarCoefficients &p1, const PolarCoefficients &p2); // all 3 coeffs equal between the 2 polars
+  void Copy(PolarCoefficients &dest, const PolarCoefficients &src); // copy the 3 coffeicients from src to dest
+
+  // remember the settings to be able to repeat the most recent values upon request
+  double _bugs = 1;
+  bool   _bugs_valid = false;
+  double _mc = 1;
+  bool   _mc_valid = false;
+  double _overload = 1;
+  bool   _overload_valid = false;
+  PolarCoefficients _ideal_polar; 
+  bool _ideal_polar_valid = false;
+  PolarCoefficients _real_polar; 
+  bool _real_polar_valid = false;
 };
 
+bool
+OpenVarioDevice::IsEqual(const PolarCoefficients &p1, const PolarCoefficients &p2)
```

Comment:
Since `IsEqual` is not comparing `OpenVarioDevice` instances, `IsPolarEqual` would be a more appropriate name

Reply:
The operand types make it obvious what is being compared.